### PR TITLE
Create Game and author classes

### PR DESCRIPTION
**Pull request by youmari**
## In this branch I implement issue #4 
- Create Game class with its properties and method
- Create Author class with its properties and method and link its relationship with item class

### CODE REVIEW

File: item.rb [hunk 1]
```diff
@@ -32,6 +32,11 @@ def move_to_archive
     @archived = true if can_be_archived?
   end
 
+  def add_author(author)
+    @author = author
+    @author.add_item(self) unless @author.items.include?(self)
+  end
+
```
Comment (Mosams) [inline item.rb:39]: Hello @youmari , very nice job so far, seeing the author and add_author function, I can't seem to see much difference except the method name. I think we should refactor this code so as to ensure it meets the DRY principle of good code structure

**Review by samwel-mongare — CHANGES_REQUESTED** [review]
Nice job so far @youmari ,

Nice code structure :+1: 
Nice implimentation of requirements :fire: 

There are however some suggestions that will make our code DRY and more effective. Please find them below.

Good luck and keep rocking :rocket:

File: item.rb (hunk 1 above)
Comment (youmari) [reply item.rb:39]: @Mosams  if you look at UML you'll see add_author method to implement the one to many relationship

**Review by samwel-mongare — APPROVED** [review]
Hello @youmari ,

There is nothing more to say other than its time to merge :1st_place_medal: 

keep rocking :rocket:

**Pull request merged**